### DISCUSSION TITLE
bundle using dynamic import

### DISCUSSION THREAD

**Discussion by kodai3** [vercel/next.js#13121] [Help]
i thought npm package used inside the component which should never be loaded dynamically, 
should not be bundled, but am i doing wrong or misunderstanding next/dynamic?

↓ "zengin-code" in this example
https://github.com/kodai3/dynamic-import-minimal-repo

<img width="1020" alt="スクリーンショット 2020-05-20 8 58 18" src="https://user-images.githubusercontent.com/33568829/82390830-b6b19380-9a7a-11ea-8543-ff0680e7de59.png">

**Comment by jamesmosier** [comment]
Are you wondering if...

a.) the node module **should** be in the dynamically loaded bundle?

or 

b.) the node module should **not** be in the dynamically loaded bundle?

**Reply by kodai3** [reply]
thank you for the quick replay🙏

a) and `dynamically loaded bundle` should **not** be in the first load ?

**Comment by jamesmosier** [comment]
If you import a npm dependency into a dynamically loaded bundle that dependency will be bundled inside the bundle.
